rumydata/field.py

```diff
@@ -37,10 +37,13 @@ class Field(_BaseSubject):
     :param strip: (optional) apply pre-processing to values in the field which
         applies the str.strip method, removing leading and trailing whitespaces,
         prior to checking rules.
+    :param ignore_if: (optional) specify values which will force the field to be
+        treated as an Ignore field. Can be a single value or list of values
     """
 
     def __init__(self, nullable=False, rules: list = None, **kwargs):
         self.strip = kwargs.pop('strip', None)
+        self.ignore_if = kwargs.pop('ignore_if', None)
```

Comment:
`self.ignore_if` = ''

Reply:
Establish this empty attribute here, update _BaseFile to pass the values from the file construction...

```
layout = {
    'col1': None,
    'col2': None,
    'col3': None
}

ExcelFile(layout, exceptions = {
    'col1': ['No']
})
```